### ISSUE TITLE
Perchè c'è un button con window.open e non un link?

### ISSUE THREAD

**Issue by nzambello** [RedTurtle/design-volto-theme#139]
Ho notato che c'è una cosa strana in questo template
Taggo @giuliaghisini perchè l'ha fatto lei e magari si ricorda qualcosa a riguardo
C'è una motivazione particolare?

Se è per una questone di stili, non è decisamente la strada giusta e voglio pensare che ci sia altro ma che vorrei risolvere in altro modo, perchè almeno ci vuole un attributo `title` o ARIA per indicare cosa succede se ci clicchi.
Così è negativo per tutti gli utenti e per nulla accessibile.

https://github.com/RedTurtle/design-volto-theme/blob/83a86a6384d012ea0af5724ebf2458665eb8bbe8/src/components/ItaliaTheme/Blocks/Listing/CompleteBlockLinksTemplate.jsx#L102

**Comment by nzambello** [comment]
Vedi anche https://github.com/RedTurtle/design-volto-theme/blob/master/src/components/ItaliaTheme/Blocks/Listing/SimpleCard/SimpleCardTemplateDefault.jsx#L135
https://github.com/RedTurtle/design-volto-theme/blob/master/src/components/ItaliaTheme/Blocks/Listing/SimpleCard/SimpleCardTemplateCompact.jsx#L70
Immagino ce ne siano altri
